audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/rest/HealthActuatorContractTest.java

```diff
@@ -45,31 +45,22 @@
 @ActiveProfiles({"akka-test"})
 class HealthActuatorContractTest {
 
-    @Autowired
-    private TestRestTemplate restTemplate;
-    @Autowired
-    private Map<String, AuditService> serviceMap;
-
     @Test
-     void testContextLoads() {
-       assertThat(serviceMap)
-               .hasSize(3)
-               .containsKeys("simple", "stroom", "logger");
-        assertThat(restTemplate).isNotNull();
+    void testContextLoads(@Autowired final Map<String, AuditService> serviceMap) {
+        assertThat(serviceMap).containsOnlyKeys("simple", "stroom", "logger");
     }
 
     @Test
-     void testIsUp() {
-        // Given that the service is running (and presumably healthy)
+    void testIsUp(@Autowired final TestRestTemplate restTemplate) {
```

Comment:
The other tests are not REST tests. better to inject into the method. I would have done the others, but the Akka stuff is global. Actually it would be better to split the test in two as the REST test do not need Akka directly.

Reply:
I thought about this and split the tests. The two are now more focused and much easier to read.